make.cmd

```diff
@@ -0,0 +1,59 @@
+@echo off
+
+setlocal
+
+set GOBIN=main
+set OUT=function.zip
+
+if "%1"=="" goto :BUILD_ALL
+if "%1"=="all" goto :BUILD_ALL
+if "%1"=="godev" goto :GO_DEV
+if "%1"=="clean" goto :CLEAN
+
+echo Invalid make target "%1"
+exit /b 2
+
+:BUILD_ALL
+    call :GCF_JS
+    if %errorlevel% neq 0 exit /b %errorlevel%
+
+    call :GCF_GO
+    if %errorlevel% neq 0 exit /b %errorlevel%
+
+    powershell -command "& {& 'Compress-Archive' -DestinationPath %OUT% -Path %GOBIN%,node_modules,index.js,package.json -Force}"
+
+    goto :EOF
```

Comment:
Maybe exit 0?

Reply:
I think it is preferable to preserve the error code